playout/libretime_playout/notify/main.py

```diff
@@ -16,19 +14,25 @@
 """
 
 import json
-import logging.config
 import sys
+import traceback
 from optparse import OptionParser
+from pathlib import Path
 
 # additional modules (should be checked)
 from configobj import ConfigObj
 
 # custom imports
 # from util import *
 from libretime_api_client import version1 as api_client
+from libretime_shared.logging import INFO, setup_logger
+from loguru import logger
 
-LOG_LEVEL = logging.INFO
-LOG_PATH = "/var/log/airtime/pypo/notify.log"
+# TODO: Get log settings from cli/env variables
```

Comment:
Please create an issue to track this once this PR is merged

Reply:
Ok